src/curated-corpus/components/actions/ScheduleCorpusItemAction/ScheduleCorpusItemAction.test.tsx

```diff
@@ -48,13 +48,15 @@ describe('The ScheduleCorpusItemAction', () => {
   it('completes the action successfully', async () => {
     // setting a fixed timezone since this test has a tendency of passing locally but failing
     // on CI due to server and local timezone discrepancies
-    const today = DateTime.local()
-      .setZone('America/New_York')
+    const chosenDate = DateTime.local()
+      .setZone('Europe/London')
       .plus({ days: 1 });
 
+    const chosenSurfaceGuid = 'NEW_TAB_EN_GB';
+
     mocks = [
```

Comment:
Updated this test with more meaningful variable names (since now() + one day was never "today" - this was a remnant of the very first iteration of this test), and a little bit more customisation of the mocks.

Also tried experimenting with setting a time zone in in CircleCI jobs, but it didn't make any difference, so I reverted those changes.